bubblewrap.c

```diff
@@ -1476,6 +1495,9 @@ main (int    argc,
       /* We don't need any caps in the launcher, drop them immediately. */
       drop_caps ();
 
+      if (opt_block_fd != -1)
+        close (opt_block_fd);
```

Comment:
Technically this is not needed, as monitor_child() closes every fd it doesn't need.